src/main/kotlin/io/cloudflight/gradle/autoconfigure/AutoConfigureGradlePlugin.kt

```diff
@@ -45,6 +45,13 @@ class AutoConfigureGradlePlugin : Plugin<Project> {
         }
 
         target.plugins.apply(ReportConfigurePlugin::class.java)
+
+        target.afterEvaluate {
+            for (child in target.subprojects) {
+                child.group = target.group
```

Comment:
we should put that functionality to the docs as well

and what if there is the rare case where someone does *not* want this behaviour? i.e. having a different groupId in a subModule?

shouldn't we just set group and version to the subprojects if there is no specified value yet?